src/alerts_utils.cc

```diff
@@ -1556,6 +1549,8 @@ alerts_utils_test(bool verbose) {
             zlist_destroy(&actions5);
         if (NULL != actions6)
             zlist_destroy(&actions6);
+
+        zsys_file_delete ("./test_state_file");
```

Comment:
`./`? There is `SELFTEST_DIR_RW` for such data items ;)

Reply:
It was done this way before, and I know this is wrong, but this is not the right time to be fixing small issues :-(

Reply:
ok